src/table/thead.tsx

```diff
@@ -46,7 +46,6 @@ export interface TheadProps {
 const Thead = React.forwardRef(
   (
     {
-      containerWidth,
```

Comment:
I guess no need to path it as a prop here
https://github.com/cloudscape-design/components/blob/8c93f3423013523ec63f20af7c6587f2bc013aa9/src/table/internal.tsx#L211